CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 - fix: SentryOptions initWithDict type errors (#1443)
 - fix: Transaction default status should be OK (#1439)
+- fix: Swizzling rootViewController from UIScenes (#1427)
```

Comment:
Way better 👏

Reply:
We should add a new entry `# Unreleased` and this item under it.
This is changing a released version